### ISSUE TITLE
Please add PR curve summaries for TF 2.0

### ISSUE THREAD

**Issue by lzvoyager** [tensorflow/tensorboard#4273]
Waymo ML platform wants to move to TF2.x. We are attempting to subclass keras.metrics.Metric for plotting PR curves. And the summary writing part is the only part that is not TF2, which makes code hard to read.

**Comment by rmothukuru** [comment]
@lzvoyager,
Can you please confirm if you have gone through [Summary_Writer API](https://www.tensorflow.org/api_docs/python/tf/summary/SummaryWriter) in `Tensorflow 2.0`? Thanks!

**Comment by lzvoyager** [comment]
Yes - I have gone through TF 2.x Summary API docs.

Internal to google, I am voyager@google.com. Also followed up on this YAQS
thread
<https://groups.google.com/a/google.com/g/tensorboard-team/c/jKPpTUIRLVE/m/UfYovsgPCgAJ>
.

**Comment by nfelt** [comment]
Duplicate of #2902

**Comment by nfelt** [comment]
I realized we already have #2902 for this - closing this one since that issue predates this one, please follow along there.
